src/Core/Checkout/Document/Renderer/RenderedDocument.php

```diff
@@ -9,8 +9,14 @@
 #[Package('checkout')]
 final class RenderedDocument extends Struct
 {
+    final public const PDF_CONTENT_TYPE = 'application/pdf';
+
+    final public const HTML_CONTENT_TYPE = 'text/html';
+
     private string $content;
 
+    private RenderedDocument $htmlA11y;
```

Comment:
```suggestion
    private ?RenderedDocument $htmlA11y = null;
```

Otherwise, we may get this error `Typed property must not be accessed before initialization` when  call `getHtmlA11y()` before  `setHtmlA11y()`